packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -193,6 +195,37 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
     }
   }
 
+  /**
+   * Resizes an image on client side, if larger than the IMG_MIN_SIZE_TO_COMPRESS env variable
+   * @param file
+   */
+  async function resizeImage(file: File): Promise<File> {
+    const imgMinSizeToCompress: number = getImgMinSizeToCompress()
+    // only resize image if larger than IMG_MIN_SIZE_TO_COMPRESS env variable
+    // ATTENTION: Media server must be configured to accept the IMG_MIN_SIZE_TO_COMPRESS
```

Comment:
Hi Michael! 

Regarding this comment - why do we nee that on the Media server? To clarify intention: I understand this code is fully browser-based and mentioned env var needs to be set on the Editor app, right?

Reply:
Your absolutely right. This code is fully browser based. If I'm remembering right, this comment is Nico's. And his intention is probably the following: If you configure the IMG_MIN_SIZE_TO_COMPRESS larger than the upload max on the server side, the server will refuse the upload and throw an error. So theses settings should be in sync for the best.